SessionControl/README.md

```diff
@@ -0,0 +1,248 @@
+# NAME
+
+**library(ses/basic)**
+
+# DESCRIPTION
+
+A library providing functions to support multiple sessions control.
+
+# VARIABLES
+
+- **sesID**
+
+    An array holding currently open session IDs. Sessions are strored from index 1,
+    index 0 is always used for the "default" **ID**. The default **ID** is always reset
+    to the last used _ID_.
+
+- **sesRunTimeout**
```

Comment:
Shall the variables be in uppercase letters? At the first sight this a bit confusing to me.

Reply:
right, probably `sesRunTIMEOUT` might be better